Makefile

```diff
@@ -2,7 +2,7 @@ include Makefile.build
 CURRENT_USER := $(shell id -u -n)
 MINIO_TAG ?= RELEASE.2021-09-15T04-54-25Z
 POSTGRES_ALPINE_TAG ?= 10.18-alpine
-DEX_TAG ?= v2.28.1
+DEX_TAG ?= v2.30.0
```

Comment:
Isn't this tag suppose to come from the environment file now? This is just the fallaback, correct?

Reply:
yes, fallback

Reply:
@DanStough the tag is exported at the root, this is a fallback like @emosbaugh says.

Reply:
i do think it would be good to change that to `latest` to avoid confusion. not donna do it in this pr